docs/monitoring/monitoring.md

```diff
@@ -3,121 +3,100 @@
+## How to Config Monitoring Settings
 
-The following two components `resources settings` are available:
+_Available as of v1.0.1_
 
-(1) Monitoring-Prometheus
+Monitoring has several components that help to collect and aggregate metric data from all Nodes/Pods/VMs. The resources required for monitoring are depends on your workloads and hardware resources. Harvester sets defaults based on general use case, and you can change them accordingly.
 
-(2) Monitoring-Prometheus-node-exporter
+Currently, `Resources Settings` can be configured for the following components:
 
-### Change resources settings of Monitoring-Prometheus
+- Prometheus
+- Prometheus Node Exporter(_UI configurable as of v1.0.2_)
 
-#### From WebUI
+### From WebUI
 
-In the `Advanced Settings` page, you can view and change the resources settings as follow:
+In the `Advanced Settings` page, you can view and change the resources settings as follows:
 
-(1) Navigate to settings page, find `harvester monitoring`.
+1. Navigate to settings page, find `harvester-monitoring`.
 ![](./assets/monitoring-setting.png)
 
-(2) Click `Show harvester-monitoring` to view the current values.
-![](./assets/monitoring-setting-view-current.png)
+1. Click `Show harvester-monitoring` to view the current values.
 
-(3) Click in the up-right corner pop up menu and select `Edit Setting` to set a new value.
+1. Click `⋮ > Edit Setting` to set a new value.
 ![](./assets/monitoring-setting-edit-config.png)
 
-(4) Click `Save`. The `Monitoring` will be restarted with the new resources settings. Please note, the restart can take some time.
+1. Click `Save` and the Monitoring will be restarted within a few seconds. Please be aware that the reboot can take some time to reload previous data.
 
-The most frequently used option is memory setting.
+**The most frequently used option is the memory setting:**
 
-`Requested Memory` is the minimum memory of the `Monitoring`. The recommended value is about 5% to 10% of system memory of one single management node. A value less than 500Mi will be denied.
+- The `Requested Memory` is the minimum memory of the `Monitoring`. The recommended value is about 5% to 10% of the system memory of one single management node. A value less than 500Mi will be denied.
 
-`Memory Limit` is the maximum memory of the `Monitoring`. The recommended value is about 30% of system memory of one single management node, when the `Monitoring` reaches this value, it will be restarted.
+- The `Memory Limit` is the maximum memory of the `Monitoring`. The recommended value is about 30% of system memory of one single management node, when the `Monitoring` reaches this threshold, it will automatically restart.
```

Comment:
```suggestion
- The `Memory Limit` is the maximum memory that can be allocated to a `Monitoring` resource. The recommended value is about 30% of the system's memory for one single management node. When the `Monitoring` reaches this threshold, it will automatically restart.
```